pixbuf-randomly-modified tries to allocate 170MB which fails in CI

--- a/tests/meson.build
+++ b/tests/meson.build
@@ -79,7 +79,6 @@ installed_tests = {
   },
   'pixbuf-fail': { 'suites': ['conform', 'slow'], },
   'pixbuf-icon-serialize': { 'suites': ['conform'], },
-  'pixbuf-randomly-modified': { 'suites': ['slow'], },
   'pixbuf-threads': { 'suites': ['io'], },
   'pixbuf-gif': {
     'suites': ['io'],
